src/Commands/Site/ImportCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Pantheon\Terminus\Commands\Site;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Terminus\Exceptions\TerminusException;
+
+class ImportCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+    /**
+     * Imports a site archive onto a Pantheon site
+     *
+     * @authorized
+     *
+     * @name site:import
+     * @alias import
+     *
+     * @option string $site Name of the site to import to
+     * @option string $url  URL at which the import archive exists
+     * @usage terminus import --site=<site_name> --url=<archive_url>
+     *   Imports the file at the archive URL to the site named.
+     */
+    public function import($sitename, $url)
+    {
+        $site = $sitename.'.dev';
+        list(, $env) = $this->getSiteEnv($site);
```

Comment:
@marktheunissen This command needs to be hard-coded to the dev environment and is not using the same workflow as that view. This command is operating SSM's `do_workflow`.